### PR TITLE
fabric: update controller deployment to enable versioned hosts

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

File: cluster/manifests/fabric-gateway/deployment.yaml [hunk 1]
```diff
@@ -48,6 +48,7 @@ spec:
             - -predicates=False()
             - -update-fabric-gateway-status=false
 {{ end }}
+            - -versioned-hosts-base-domain={{ .Values.hosted_zone }}
```
Comment (szuecs) [inline cluster/manifests/fabric-gateway/deployment.yaml:51]: does this include `ingress.cluster.local`zone?

Comment (AlexanderYastrebov) [reply cluster/manifests/fabric-gateway/deployment.yaml:51]: No

Comment (szuecs) [reply cluster/manifests/fabric-gateway/deployment.yaml:51]: Then we want to have a list here and include it, I guess.

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**